mne/evoked.py

```diff
@@ -168,7 +173,12 @@ def apply_baseline(self, baseline=(None, 0), verbose=None):
 
         .. versionadded:: 0.13.0
         """
+        _check_baseline(baseline, self.times[0], self.times[-1],
+                        self.info['sfreq'])
+        # Rescale actually only happens if baseline is not None.
         self.data = rescale(self.data, self.times, baseline, copy=False)
+        if baseline is not None:
+            self.baseline = baseline
```

Comment:
can you check that one cannot pass baseline None if self.baseline is not None? basically once a baseline has been used you cannot go back

Reply:
That's effectively what's happening, actually. If `baseline=None`, `rescale` won't do anything. I also just copied the behavior from `Epochs` here. I could re-arrange the code such that things become clearer:

```python
        if baseline is not None:
            self.data = rescale(self.data, self.times, baseline, copy=False)
            self.baseline = baseline
```
WDYT?

Reply:
I've addressed this in 8ee3bb7